lsp.client/src/org/netbeans/modules/lsp/client/ConnectToLanguageServer.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.lsp.client;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.File;
+import org.openide.*;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionRegistration;
+import org.openide.filesystems.FileObject;
+import org.openide.filesystems.FileUtil;
+import org.openide.util.NbBundle.Messages;
+
+@ActionID(
+        category = "Tools",
+        id = "org.netbeans.modules.lsp.client.ConnectToLanguageServer"
+)
+@ActionRegistration(
+        displayName = "#CTL_ConnectToLanguageServer"
+)
+@Messages("CTL_ConnectToLanguageServer=Connect to Language Server")
```

Comment:
I think a natural place for this action could be a pop-up menu on the non-recognized data objects. E.g. if the IDE doesn't know a `.kt` file it's popup menu would suggest to connect to a language server.